src/com/lftechnology/java/training/niraj/ems/services/implementations/UserServiceImpl.java

```diff
@@ -0,0 +1,507 @@
+package com.lftechnology.java.training.niraj.ems.services.implementations;
+
+import java.io.Console;
+import java.sql.SQLException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.niraj.ems.domains.Employee;
+import com.lftechnology.java.training.niraj.ems.enums.Operators;
+import com.lftechnology.java.training.niraj.ems.enums.Roles;
+import com.lftechnology.java.training.niraj.ems.enums.Status;
+import com.lftechnology.java.training.niraj.ems.exceptions.CustomException;
+import com.lftechnology.java.training.niraj.ems.services.CrudService;
+import com.lftechnology.java.training.niraj.ems.utils.Constants;
+import com.lftechnology.java.training.niraj.ems.utils.UserNumInputImpl;
+import com.lftechnology.java.training.niraj.ems.utils.UserStringInputImpl;
+import com.lftechnology.java.training.niraj.ems.utils.Utils;
+
+public class UserServiceImpl implements CrudService<Employee, String> {
+
+    private static final Logger LOGGER = Logger.getLogger(UserServiceImpl.class.getName());
+    private EmployeeDaoImpl employeeDao;
+    private UserStringInputImpl userInput;
+    private UserNumInputImpl userNumInput;
+
+    public UserServiceImpl() {
+        employeeDao = new EmployeeDaoImpl();
+        userInput = new UserStringInputImpl();
+        userNumInput = new UserNumInputImpl();
+    }
+
+    @Override
+    public Employee save(Employee t) throws SQLException {
+        return employeeDao.save(t);
+    }
+
+    @Override
+    public Employee find(String s) throws SQLException {
+        return employeeDao.find(s);
+    }
+
+    @Override
+    public boolean update(Employee t) throws Exception {
+        return employeeDao.update(t);
+
+    }
+
+    /**
+     * Logs a user in
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param username
+     * @param password
+     * @return
+     * @throws SQLException
+     */
+    public Employee login(String username, String password) throws SQLException {
+        Map<String, String> credentials = new LinkedHashMap<String, String>();
+        credentials.put("username", username);
+        credentials.put("password", password);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "password" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)